### DISCUSSION TITLE
What will happen if my computing power is lost for several days?

### DISCUSSION THREAD

**Discussion by FreeBoys** [filecoin-project/lotus#8255] [Storage Provider] [answered]
What will happen if my computing power is lost for several days?

**Comment by Reiers** [comment]
Moving to Discussions.

**Comment by Reiers** [comment, marked as answer]
After 48 hours from the first "lost" windowPost you will start pay penalties. 

https://docs.filecoin.io/storage-provider/slashing/
-
https://docs.filecoin.io/reference/glossary/#slash
